source/adware/wildcard.list

```diff
@@ -1,8 +1,14 @@
+girlsoncam.info
+grtyv.com
+hrtya.com
+jizzy.org
+sexemulator.com
+irtyf.com
 1bcde.com
 1ccbt.com
 1s3h4f7sl.com
 360.com
-360safe.com�.cn
+360safe.comð.cn
```

Comment:
Should be `360safe.xn--com-5ma.cn` (https://en.wikipedia.org/wiki/Punycode)

No ?

Reply:
:rofl: that's called a typo :smiling_imp: